owiddata/owiddata-stats.json

```diff
@@ -5,29 +5,29 @@
   "Sinovac_country_count": "63",
   "owid_RNA_count": "2",
   "owid_RNA_countries": "168",
-  "owid_RNA_map": "https://github.com/greenelab/covid19-review/raw/94d16532b2ab833c40757b7807851db351668eed/owiddata/maps/RNA.png",
+  "owid_RNA_map": "https://github.com/greenelab/covid19-review/raw/$FIGURE_COMMIT_SHA/owiddata/maps/RNA.png",
   "owid_VLP_count": "1",
   "owid_VLP_countries": "1",
-  "owid_VLP_map": "https://github.com/greenelab/covid19-review/raw/94d16532b2ab833c40757b7807851db351668eed/owiddata/maps/VLP.png",
-  "owid_commit_date": "2022-08-08T07:06:25Z",
-  "owid_commit_date_pretty": "August 8, 2022",
-  "owid_commit_sha": "a08e593a427426fc1129a98609ef4d52344fb7de",
-  "owid_doses_bargraph": "https://github.com/greenelab/covid19-review/raw/94d16532b2ab833c40757b7807851db351668eed/OWID_doseTypebyGDP.png",
+  "owid_VLP_map": "https://github.com/greenelab/covid19-review/raw/$FIGURE_COMMIT_SHA/owiddata/maps/VLP.png",
+  "owid_commit_date": "2022-08-09T11:25:42Z",
+  "owid_commit_date_pretty": "August 9, 2022",
+  "owid_commit_sha": "13ee8fc7b087b6195f5a2fab909a976057d0c973",
+  "owid_doses_bargraph": "https://github.com/greenelab/covid19-review/raw/$FIGURE_COMMIT_SHA/OWID_doseTypebyGDP.png",
   "owid_inactivated_count": "10",
   "owid_inactivated_countries": "116",
-  "owid_inactivated_map": "https://github.com/greenelab/covid19-review/raw/94d16532b2ab833c40757b7807851db351668eed/owiddata/maps/inactivated.png",
-  "owid_most_recent_date": "August 7, 2022",
+  "owid_inactivated_map": "https://github.com/greenelab/covid19-review/raw/$FIGURE_COMMIT_SHA/owiddata/maps/inactivated.png",
+  "owid_most_recent_date": "August 8, 2022",
   "owid_non_replicating_viral_vector_count": "5",
   "owid_non_replicating_viral_vector_countries": "202",
-  "owid_non_replicating_viral_vector_map": "https://github.com/greenelab/covid19-review/raw/94d16532b2ab833c40757b7807851db351668eed/owiddata/maps/non_replicating_viral_vector.png",
-  "owid_plasmid_vectored_count": "0",
-  "owid_plasmid_vectored_countries": "0",
+  "owid_non_replicating_viral_vector_map": "https://github.com/greenelab/covid19-review/raw/$FIGURE_COMMIT_SHA/owiddata/maps/non_replicating_viral_vector.png",
   "owid_protein_subunit_count": "10",
   "owid_protein_subunit_countries": "41",
-  "owid_protein_subunit_map": "https://github.com/greenelab/covid19-review/raw/94d16532b2ab833c40757b7807851db351668eed/owiddata/maps/protein_subunit.png",
+  "owid_protein_subunit_map": "https://github.com/greenelab/covid19-review/raw/$FIGURE_COMMIT_SHA/owiddata/maps/protein_subunit.png",
   "owid_total_countries": "223",
-  "owid_total_vaccinations": "12.0 billion",
   "owid_vaccine_counts": "29",
+  "owid_worldwide_daily_rate": "808.0 per million",
+  "owid_worldwide_date": "August 2, 2022",
+  "owid_worldwide_total_vaccinations": "12.0 billion",
```

Comment:
Will we need to check in the manuscript if this variable has a valid value or a message like "Cannot convert count N/A to billions"?  I'm wondering if this message is going to be read by maintainers, in which case it looks good, or by a downstream Jinja conditional statement, in which case we can simplify it.